pkg/binutils/binaries.go

```diff
@@ -29,6 +29,7 @@ var (
 
 type PluginBinaryDownloader interface {
 	Download(vmIDs map[string]struct{}, pluginDir, binDir string) error
+	DownloadVM(vmID string, pluginDir, binDir string) error
```

Comment:
why not use Download?

Reply:
Download has a lot of additional complexity. Honestly, I had a lot of confusion trying to figure out how to use it. DownloadVM was much simpler and already public, just needed to be added to the interface.